faqs/stocks/which-screener.md

```diff
@@ -9,75 +9,122 @@ description: >-
 
 ## Introduction
 
-Even though screeners are useful in finding out the investment opportunities which suits your criteria and to monitor the existing investment, it might not be a good idea to take a screener at its face value.
+Though stock-screeners calculate a lot of things, you must do you own calculations too. You will miss a lot of good opportunities or make some expensive mistakes if you consider only the pre-calculated values.
 
-You might see an opportunity or a red-flag from these screeners. However, it could easily just be a calculation error, or usage of incorrect formula, or not removing an outlier.
+The objective of this article is not to demean the websites but to show the importance of your own calculations.
 
-This article is to sensitize you that you might have to do more research rather than just going with the presented data from the UI of a screener.
+----
 
-## Examples of incorrect data in screeners
+## Examples of dividend yield
 
-We've come across some incorrect computations and results that are outright wrong. We've also informed the teams behind these screeners. At the time of writing this, these have not been addressed.  
-  
-It's our duty to warn others to be mindful of these, while they embark on their research.
+Lets consider the dividend yield of Britannia Industries as on 29th April 2021:
 
-Following are a few examples of such issues we'd noticed when we were doing this research into existing screeners.
+```
+Screener.in:    4.18%
+MoneyControl:   0.95%
+MorningStar.in: 1.78%
 
-### Gross Profit Margin
+Correct value:  ~2-5%
```

Comment:
>  Correct value:  ~2-5%

Not sure _correct value_ is the right characteriztion. Perhaps we should mention under which condition, the dividend yield would ideally range from ~2-5%.

Reply:
I don't know. May be we can write it as *right value*.

I think the current format communicates the purpose. That the correct value is subjective and depends on ones estimate how how repeatable these dividends are.

Company's usually classify non-repeatable (one-time) dividends as special dividends. However, in both the case of Britannia and Majesco, the companies classified them as normal or interim dividends. This is the reason portals are unable to do anything.